### PR TITLE
Helm Release Notes for 4.11

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
[RHDEVDOCS-4088](https://issues.redhat.com//browse/RHDEVDOCS-4088): Document Helm Release Notes for 4.11

- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.11`
- **JIRA issues**: [RHDEVDOCS-4088](https://issues.redhat.com/browse/RHDEVDOCS-4088)
- **Preview pages**: [Download to see the preview in your browser](https://drive.google.com/file/d/1MdW6ldaZ7kI4ny6-Hd9u2TDgT8BQu60Q/view?usp=sharing)
- **SME Review**: Completed by @dperaza4dustbit, @Kartikey-star 
- **QE review**:  Completed by @tisutisu
- **Peer-review**: Completed by @rolfedh

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@dperaza4dustbit, @Kartikey-star, and @tisutisu: PTAL at the PR and provide your feedback. Thanks!

**Review by Kartikey-star — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1283,6 +1283,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
 
+* There is currently a known issue when a user is creating an instance of ProjectHelmChartRepository custom resource (CR) or HelmChartRepository CR with TLS verification. As a result, the user cannot list the repositories and perform Helm-related operations with the ProjectHelmChartRepository and HelmChartRepository that supports TLS verification. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
```
[the remark this answers is not in the document]
Comment (dperaza4dustbit) [reply release_notes/ocp-4-11-release-notes.adoc]: Is this only for ProjectHelmChartRepository or also for HelmChartRepository @Kartikey-star

[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply release_notes/ocp-4-11-release-notes.adoc]: Done.

**Comment by Srivaralakshmi** [comment]
@Kartikey-star Thanks for the approval.

@dperaza4dustbit and @tisutisu Please review and provide feedback.

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1330,6 +1330,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
 
+* There is currently a known issue when a user is creating an instance of ProjectHelmChartRepository custom resource (CR) with TLS verification. As a result, the user cannot list the repositories and perform Helm-related operations with the ProjectHelmChartRepository that supports TLS verification. There is currently no workaround for this issue.(link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
```
Comment (rolfedh) [inline release_notes/ocp-4-11-release-notes.adoc:1333]:
```suggestion
* There is currently a known issue. If you create an instance of `ProjectHelmChartRepository` custom resource (CR) that uses TLS verification, you cannot list repositories and perform Helm-related operations. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
```

Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc:1333]: Please consider making these changes.

**Review by rolfedh — CHANGES_REQUESTED** [review]

**Comment by tisutisu** [comment]
LGTM

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply release_notes/ocp-4-11-release-notes.adoc:1333]: Done.

**Review by rolfedh — APPROVED** [review]

**Pull request merged**